daemon/algod/server.go

```diff
@@ -130,6 +130,20 @@ func (s *Server) Initialize(cfg config.Local, phonebookAddresses []string, genes
 	if err != nil {
 		return fmt.Errorf("Initialize() err: %w", err)
 	}
+	// TODO: remove this after making pebble support official
+	// and integrate the value into ReservedFDs config parameter.
+	if cfg.StorageEngine == "pebbledb" {
+		fdRequired = ot.Add(fdRequired, 1000)
+		if ot.Overflowed {
+			return errors.New(
+				"Initialize() overflowed when adding up fdRequired and 1000 needed for pebbledb decrease them")
```

Comment:
"and 1000 needed for pebbledb decrease them"

Not sure what "decrease them" here means, suggested to remove so it just says

"and 1000 needed for pebbledb"